src/solvers/dg_structured/containers_3d.jl

```diff
@@ -0,0 +1,68 @@
+function init_elements!(elements, mesh::StructuredMesh{3, RealT}, nodes) where {RealT}
+  n_nodes = length(nodes)
+
+  @unpack coordinates_min, coordinates_max, linear_indices = mesh
+
+  # Get cell length
+  dx = (coordinates_max[1] - coordinates_min[1]) / size(mesh, 1)
+  dy = (coordinates_max[2] - coordinates_min[2]) / size(mesh, 2)
+  dz = (coordinates_max[3] - coordinates_min[3]) / size(mesh, 3)
+
+  inverse_jacobian = 8/(dx * dy * dz)
+
+  # Calculate inverse Jacobian and node coordinates
+  for element_x in 1:size(mesh, 1), element_y in 1:size(mesh, 2), element_z in 1:size(mesh, 3)
```

Comment:
To simplify the many cases where you iterate over the cells in a dimension by dimension, you could consider adding
```julia
Base.axes(mesh::StructuredMesh) = map(Base.OneTo, mesh.cells_per_dimension)
Base.axes(mesh::StructuredMesh, i) = Base.OneTo(mesh.cells_per_dimension[i])
```
to `src/mesh/structured_mesh.jl` and then iterate over `cell_x in axes(mesh, 1)` etc.?

Reply:
In some cases we iterated over `2:size(mesh, 1)`, so we would still need to use `size(mesh)`. Wouldn't it be more consistent to always use size?

Reply:
Yes, also a good point. Just leave it as it is for now then.